posthog/settings/data_stores.py

```diff
@@ -126,6 +126,8 @@
 KAFKA_SASL_USER = os.getenv("KAFKA_SASL_USER", None)
 KAFKA_SASL_PASSWORD = os.getenv("KAFKA_SASL_PASSWORD", None)
 
+KAFKA_DEFAULT_TOPIC_REPLICATION_FACTOR = os.getenv("KAFKA_DEFAULT_TOPIC_REPLICATION_FACTOR", 1)
```

Comment:
This feels fundamentally dangerous to specify this way - it should lead to installations with insufficient replication factors.

Reply:
How else should this be specified? Currently all topics are autocreated and I believe with a replication factor of 1.

Somewhere this needs to be a config option, even if not here